examples/kitchensink/src/admin/extensions/theme.ts

```diff
@@ -0,0 +1,48 @@
+const colors: object = {
```

Comment:
Sometimes you're using `object` & other times you're using `{}`, is there any reason?

Reply:
Yes for the moment I was trying to check if it was throwing an error correctly this file is going to be removed soon.